### ISSUE TITLE
Choose tab: add map and store data in temporary reactiveValues

### ISSUE THREAD

**Issue by elma13** [rivm-syso/Analyse-Together#397]
Do the part of the map and store to get the data in the START-page only once the data in a new issue.

_Originally posted by @elma13 in https://github.com/rivm-syso/Analyse-Together/issues/391#issuecomment-2449936563_
